### PR TITLE
url-tabs

### PR DESCRIPTION
**Pull request by San88Francisco**
https://app.clickup.com/t/869589070

it is commented out there, it should be so, if uncommented then useSearchParams works globally

> - created/ accordion and tabs components
> - coments/ useSearchParams in modalPortal and headerListIcons
> - created/ file components put accordions
> - created/ a11Tabs, accordions-panel and tab-panel utils
- deleted/ style in account-page

### CODE REVIEW

File: src/components/Layout/header/header-list-icons/headerListIcons.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* eslint-disable prettier/prettier */
```
Comment (pqoqubbw) [inline src/components/Layout/header/header-list-icons/headerListIcons.tsx]: remove this line

File: src/components/Layout/header/header-list-icons/headerListIcons.tsx [hunk 2]
```diff
@@ -18,13 +18,13 @@ const HeaderListIcons = () => {
...
 
-  const searchToken = Object.fromEntries(searchParams.entries())
+  // const searchToken = Object.fromEntries(searchParams.entries())
 
-  useEffect(() => {
-    Object.keys(searchToken).length > 0 && setIsModalOpen(true)
-  }, [searchToken])
+  // useEffect(() => {
+  //   Object.keys(searchToken).length > 0 && setIsModalOpen(true)
+  // }, [searchToken])
```
Comment (pqoqubbw) [inline src/components/Layout/header/header-list-icons/headerListIcons.tsx]: pull changes from `master`

File: src/components/modal-portal/ModalPortal.tsx [hunk 3]
```diff
@@ -19,33 +19,32 @@ type Props = {
...
+  //   dispatch(resetAuth())
+  //   if (searchToken) {
+  //     const searchKey = Object.keys(searchToken)[0]
+  //     searchParams.delete(searchKey)
+  //     navigate({ search: searchParams.toString() })
+  //     setIsModalOpen(false)
+  //     return
+  //   }
+  //   setIsModalOpen(false)
+  // }
```
Comment (pqoqubbw) [inline src/components/modal-portal/ModalPortal.tsx]: ?

File: src/pages/account-page/account-page.tsx [hunk 4]
```diff
@@ -1,225 +1,32 @@
...
-    setActiveIndex(newValue)
-  }
+  const initialTabIndex = searchParams.get('tab')
+    ? parseInt(searchParams.get('tab') || '1', 10) - 1
+    : 0
 
-  const handleAccordionChange = (newIndex: number | null) => {
-    setActiveIndex(newIndex ?? 0)
-  }
+  const [activeIndex, setActiveIndex] = useState(initialTabIndex)
```
Comment (pqoqubbw) [inline src/pages/account-page/account-page.tsx]: why do you need this state if you alreade have index in `url`?

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

File: src/components/Layout/header/header-list-icons/headerListIcons.tsx (hunk 1 above)
Comment (San88Francisco) [reply src/components/Layout/header/header-list-icons/headerListIcons.tsx]: done

File: src/components/Layout/header/header-list-icons/headerListIcons.tsx (hunk 2 above)
Comment (San88Francisco) [reply src/components/Layout/header/header-list-icons/headerListIcons.tsx]: done

**Review by pqoqubbw — APPROVED** [review]

File: src/components/modal-portal/ModalPortal.tsx (hunk 3 above)
Comment (San88Francisco) [reply src/components/modal-portal/ModalPortal.tsx]: done

File: src/pages/account-page/account-page.tsx (hunk 4 above)
Comment (San88Francisco) [reply src/pages/account-page/account-page.tsx]: remove

**Pull request merged**